cpp/trtorchc/main.cpp

```diff
@@ -375,11 +402,6 @@ int main(int argc, char** argv) {
     return 1;
   }
 
-  if (!trtorch::CheckMethodOperatorSupport(mod, "forward")) {
```

Comment:
Why was this removed?

Reply:
Fixed it now. CheckMethodOperatorSupport errors out listing all the unsupported ops. When torch_fallback is enabled, this will stop execution without even going to CompileGraph which shouldn't happen.